### ISSUE TITLE
The problem of computing light-curve after upgrading

### ISSUE THREAD

**Issue by bluecandyrain** [ranieremenezes/easyfermi#53]
When it computes the LC, it gets stuck at the process of 80%. I check the files and find that the LC files of each bin have been computed completely, but it can't product the LC file. I also check the task manager and find that the CPU dose not works at all.

easyfermi: Version 2.0.10.
OS: I try it in both Linux and Windows system and encounter the same problem.

**Comment by ranieremenezes** [comment]
Hello, please tell me what is the configuration you are using, as the target name/coordinates, energy range, time range etc, so I can try to reproduce the error. 

Furthermore, do you have errors in the terminal or easyfermi log? Did you try running easyfermi with other configurations?

Please attach also the fermipy.log file

**Comment by bluecandyrain** [comment]
OK. The target name is 4FGL J0538.8-4405. 
The configuration and log files:
[config.txt](https://github.com/ranieremenezes/easyfermi/files/14532027/config.txt)
[fermipy.log](https://github.com/ranieremenezes/easyfermi/files/14532031/fermipy.log)

In fact, the easyfermi and terminal window have never pop up any error and just stay at the 'computing light curve'. 

Before this I have already try another configuration of another source in Ubuntu （and this is in WSL）, it's the same problem. But the files were deleted because that computer is not mine and the disk space is not enough.

**Comment by ranieremenezes** [comment]
Hello,

I also need the information in the easyfermi window. For instance, how many bins are you trying? What is the fit configuration? I see in the fermipy.log file the following message: 
_Fit failed in bin 573 in range 390441601 390700801._

I guess you asked for more than 600 bins. In this case: 
1) the process will take forever.
2) You basically have no photons to perform the fit. So I would not trust the results.

I see you are using a very short energy range from 100 to 300 MeV. This is really short! For such a case I would recommend using the custom mode in easyfermi and increase binsperdec in the config file.

Furthermore, you said that you tested other configuration. Which configuration was it? It would be very helpful to have the exact configuration and possibly a print screen of the easyfermi window.

**Comment by bluecandyrain** [comment]
OK, this is the easyfermi window:
![easyfermi](https://github.com/ranieremenezes/easyfermi/assets/148299198/275834ce-cc26-4519-bf6a-8c4252b4cbda)

The bins are really large, but I have tried 650 bins before upgrading and it worked out successfully, so I guess it could make it in this version. If it is the crux of the problem, I will correct it. 
About the energy range I just want to try it and see if the significance is acceptable. If it is out of the lines of LAT data, I also correct it. By the way, I have got some 100-300MeV data of sources which are at meddle bright level, the TS and error level are really terrible.
Another configuration is similar to this one, the difference point is just the coordinate.
